importer/importer.py

```diff
@@ -252,4 +252,4 @@ def main(arguments):
     parser.add_argument("--table", action='store_true')
     args = parser.parse_args()
     print(args)
-    main(args)
+    main(args)
```

Comment:
Add a newline to the end of the file (not strictly needed but expected). Not sure why it disappeared.